### PR TITLE
Adding top 5 songs

### PR DESCRIPTION
**Pull request by morjam22**
@mjfaade10 how do you feel about these top 5 rappers?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers
```
Comment (mjfaade10) [inline README.md]: Can you change this to Top 5 rappers of 2020?

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+
+1. JCole
+2. Kendrick Lamar
```
Comment (mjfaade10) [inline README.md]: Should we replace Kendrick? He hasn't come out with an album in years.

**Review by mjfaade10 — CHANGES_REQUESTED** [review]
I added a couple of updates

File: README.md (hunk 1 above)
Comment (morjam22) [reply README.md]: This has been updated

File: README.md (hunk 2 above)
Comment (morjam22) [reply README.md]: Good point. I replaced Kendrick with Travis. Let me know what you think!

File: README.md [hunk 3]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers of 2020
+
+1. JCole
+2. Travis Scott
+3. Drake
+4. Big Sean
+5. Dom Kennedy
```
Comment (mjfaade10) [inline README.md]: I think Dom Kennedy should be replaced as well :/ He's got some classics, but not in 2020.

File: README.md (hunk 2 above)
Comment (mjfaade10) [reply README.md]: Good choice!

File: README.md (hunk 3 above)
Comment (morjam22) [reply README.md]: How do you feel about Meg? She's got some hits and her tiny desk performance was one of a kind.

Comment (mjfaade10) [reply README.md]: I love that idea! Go ahead and replace Dom Kennedy with Meg.

File: README.md [hunk 4]
```diff
@@ -1 +1,9 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers of 2020
```
Comment (mjfaade10) [inline README.md]: Can we actually change this to Top 5 Albums? I think that would be a list worth seeing.

File: README.md [hunk 5]
```diff
@@ -1 +1,9 @@
...
+
+1. JCole
+2. Travis Scott
+3. Drake
+4. Big Sean
+5. Meg
```
Comment (mjfaade10) [inline README.md]: We may need to replace Meg in this case :/

**Review by mjfaade10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (morjam22) [reply README.md]: Sure! I like that idea a lot.

File: README.md (hunk 5 above)
Comment (morjam22) [reply README.md]: Yea, I agree. I don't think she's going to make the cut.

File: README.md (hunk 4 above)
Comment (mjfaade10) [reply README.md:3]: One last suggestion. Let's update the title to Top 5 Albums. That may be easier :)

Comment (morjam22) [reply README.md]: Top 5 albums of 2020? Or just in general?

Comment (mjfaade10) [reply README.md]: In general sounds good. Thanks!

File: README.md [hunk 6]
```diff
@@ -1 +1,10 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Albums
+
+1. Frank Ocean - Blonde
+2. Drake - More Life
+2. Travis Scott - Astro World
+4. Kanye West - My Beautiful Dark Twisted Fantasy
```
Comment (mjfaade10) [inline README.md:8]: I'm not a huge fan of Kanye West these days, but I do love this album. Good choice.

Comment (mjfaade10) [reply README.md:8]: This is good to go!

**Pull request merged**